cas-server-support-osf/src/main/java/io/cos/cas/web/flow/OpenScienceFrameworkAuthenticationExceptionHandler.java

```diff
@@ -78,6 +79,7 @@
 
     // Customized exceptions for OSF
     static {
+        DEFAULT_ERROR_LIST.add(InvalidVerificationKeyException.class);
```

Comment:
Auth exception handler is updated  to include the new exception. Please note that invalid verification key does not raise rate-limit throttle.